rotkehlchen/api/rest.py

```diff
@@ -3929,8 +3929,10 @@ def add_manual_latest_price(
                 result=wrap_in_fail_result(message=str(e)),
                 status_code=HTTPStatus.CONFLICT,
             )
-
-        Inquirer().remove_cached_current_price_entry(cache_key=(from_asset, to_asset))
+        for pair in pairs_to_invalidate:
```

Comment:
This logic is also in `delete_manual_latest_price`. Can you maybe just consolidate it out and abstract it into a common function? Don't copy paste code around.